src/lib/products.ts

```diff
@@ -20,10 +20,10 @@ const productSlugsToNames: { [slug in ProductSlug]: ProductName } = {
  * Holds the product slugs in the order ProductSwitcher will render them.
  */
 const productSwitcherSlugs: ProductSlug[][] = [
-  ['terraform', 'packer', 'vagrant'],
-  ['vault', 'boundary'],
+  ['terraform', 'packer'],
   ['consul'],
-  ['nomad', 'waypoint'],
+  ['vault', 'boundary'],
+  ['nomad', 'waypoint', 'vagrant'],
```

Comment:
Update ordering to reflect the current groupings and order for rendering.

- Moves vagrant to be alongside nomad and waypoint
- Moves consul to be the second array within the grouping

<img width="1209" alt="image" src="https://user-images.githubusercontent.com/825855/163032604-50382c53-fe3b-44cd-86d0-a9ac95e19cd2.png">